src/components/Notification/NotificationInfo.js

```diff
@@ -0,0 +1,28 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const NotificationInfo = ({ time, date, className, ...props }) => {
+  const classes = ClassNames('drawer-pf-notification-info', className);
+
+  return (
+    <div className={classes}>
+      <span className="date">{date}</span>
+      <span className="time">{time}</span>
+    </div>
```

Comment:
This is specific to displaying the date and time. Applications may want to display other information here. Not sure if we should have a NotificationDateTimeInfo component for this (which seems common) case and have NotificationInfo be generic or handle children here and allow for date/times to be null.

Reply:
the spans need those classnames to render correctly, I can create `NotificationLeftText`,`NotificationRightText` components that will use those classnames.